packages/core/src/odata/v2/odata-batch-request-builder.ts

```diff
@@ -364,16 +362,6 @@ function getEndBatchWithLineBreak(batchId: string) {
   return `--batch_${batchId}--\n`;
 }
 
-function getMethod(
```

Comment:
I assume the methods are moved to batch utils without changes.

Reply:
yes the v2 and v4 implementation are both used to the common.